pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicyrule/computefirewallpolicyrule-egress-full/_http.log

```diff
@@ -735,13 +640,6 @@ X-Xss-Protection: 0
 {
   "error": {
     "code": 400,
-    "errors": [
-      {
-        "domain": "global",
-        "message": "Invalid value for field 'priority': '9000'. The firewall policy does not contain a rule at priority 9000.",
-        "reason": "invalid"
-      }
-    ],
```

Comment:
The errors field comes from mock memory([code](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/4e8eb92702823e8caa3abff45b6d88b37f36a117/mockgcp/pkg/storage/memory.go#L132)).

In the mock server, because rule is a virtual resource without unique identifier, I did not save it in to the mock storage. If a certain rule is not found in a policy, I log an error message directly, so it's expected to not have the extra "errors" field in the error response.